packages/main/src/security-restrictions.ts

```diff
@@ -41,12 +41,14 @@ const ALLOWED_ORIGINS_AND_PERMISSIONS = new Map<
     | 'openExternal'
     | 'window-placement'
     | 'unknown'
+    | 'https://sso.redhat.com'
   >
 >(
   import.meta.env.DEV && import.meta.env.VITE_DEV_SERVER_URL
     ? [[new URL(import.meta.env.VITE_DEV_SERVER_URL).origin, new Set(['clipboard-sanitized-write'])]]
     : [],
 );
+ALLOWED_ORIGINS_AND_PERMISSIONS.set('https://sso.redhat.com',new Set(['unknown']));
```

Comment:
it does not look following extension model compliance there

Reply:
There should be a way to open external URL, or using external window for auth is the only way to go (see previous comment) because that would not require any changes in extension API.